packages/material-ui-lab/src/SliderStyled/SliderStyled.js

```diff
@@ -0,0 +1,353 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useThemeProps, muiStyled, fade, lighten, darken } from '@material-ui/core/styles';
+import { propsToClassKey } from '@material-ui/styles';
```

Comment:
A note for another day. It sounds like we will be able to drop `@material-ui/styles` in the end.